test/instrumentation/modules/next/a-nextjs-app/.tav.yml

```diff
@@ -0,0 +1,32 @@
+# Test Next.js versions.
```

Comment:
I don't have an entirely solid answer. Mainly because it allows for clean separation of the package.json for this Next.js app. If this tav config was in the top-level .tav.yml file, then the dependencies:

```
  "dependencies": {
    "next": "^13.0.0",
    "react": "^18.2.0",
    "react-dom": "^18.2.0"
  }
```

would have to be in the top-level "devDependencies". In general that muddies the waters for understanding just this Next.js test case.  Currently as it is you can work just in this test dir:

```
cd test/instrumentation/modules/next/a-nextjs-app
npm install
node ../next.test.js
npx tav
```

While doing (the long) dev on this PR that was helpful. And especially while learning Next.js it was helpful to be clear about exactly what deps were involved for the test app... and not the ~80 devDeps that we currently have at the top-level.

In this case there aren't any module conflicts but, for example, there *are* with the graphql-related modules (hence the `npm uninstall ...` commands in .tav.yml. I'm not proposing doing it soon, but I think it would be interesting to eventually consider a layout where there are multiple separate .tav.yml files -- perhaps one per instrumentation module.